### PR TITLE
docs: added Yakuza Kiwami 2

### PR DESCRIPTION
**Pull request by Anomamu**
# Description

## Type(s) 

- [x] Game page addition
- [ ] Game page modification
- [ ] Other wiki edit

## Verification 
Have you

- [x] Followed the [contribution guidelines?](https://github.com/Whisky-App/whisky-book?tab=readme-ov-file#please-read-all-steps-before-contributing)
- [x] Ran `./scripts/generate.mjs`?
- [x] Ran `./scripts/lint.mjs`?
- [x] Checked spelling and grammar?
- [x] (If applicable) Provided documentation for a game not working? (i.e. detailing what happens when the game doesn't work?)
- [x] (If applicable) Ensured the linked Steam ID is the correct one for your game addition?

### CODE REVIEW

File: src/SUMMARY.md [hunk 1]
```diff
@@ -198,6 +198,7 @@
   - [Wolfenstein: The New Order](./game-support/wolfenstein-new-order.md)
   - [Wolfenstein: The Old Blood](./game-support/wolfenstein-old-blood.md)
   - [Wolfenstein II: The New Colossus](./game-support/wolfenstein2-new-colossus.md)
+  - [Yakuza Kiwami 2](./game-support/yakuza_kiwami_2.md)
```
Comment (mattmaniak) [inline src/SUMMARY.md:201]: Please replace underscores with dashes. As mentioned in Readme.

**Comment by mattmaniak** [comment]
Check and fill the  "Verification. Have you" section in your PR template. Checkboxes are missing.

**Review by mattmaniak — CHANGES_REQUESTED** [review]

File: src/SUMMARY.md (hunk 1 above)
Comment (Anomamu) [reply src/SUMMARY.md:201]: Changes made.

Comment (mattmaniak) [reply src/SUMMARY.md:201]: I see a change with PR checkboxes but the markdown filename remains the same (with underscores) still.

Comment (hahayupgit) [reply src/SUMMARY.md]: Manually fixed by me.

**Comment by hahayupgit** [comment]
@Anomamu I've gone in and fixed your PR, in the future please make sure to run `./scripts/generate.mjs` and `./scripts/lint.mjs` before merging your PRs to catch any errors. You had swapped the review metadata and game embed tags.

**Review by hahayupgit — APPROVED** [review]

**Review by mattmaniak — APPROVED** [review]
It seems that I missed something before... @hahayupgit
Now it's all good.

**Comment by hahayupgit** [comment]
> It seems that I missed something before... @hahayupgit Now it's all good.

It was just linting stuff, you would''ve caught it when you ran the checks 😉

**Pull request merged**